### DISCUSSION TITLE
Heatmap panel: GnYlRd color scheme

### DISCUSSION THREAD

**Discussion by acerioni** [grafana/grafana#38227] [Dashboards, Panels & Visualizations]
**What would you like to be added**: GnYlRd color scheme, or else the possibility to invert already existing color schemes

**Why is this needed**: in some application the GnYlRd color scheme may be semantically more appropriate than the already existing RdYlGn color scheme

**Comment by ivanahuckova** [comment]
Hello @acerioni, I have a good news for you, Grafana version 7.3.0+ comes with Green-Yellow-Red color scheme. See https://grafana.com/docs/grafana/latest/whatsnew/whats-new-in-v7-3/#new-color-scheme-option

**Comment by acerioni** [comment]
Hello @ivanahuckova, thanks for your kind reply. Unfortunately, I don't see how to use this new feature. I am styling a heatmap with Grafana version 7.4.0. Thanks in advance for your help!

**Comment by ivanahuckova** [comment]
@acerioni sorry, I didn't realize you were talking about heatmaps panel as we have recently added color scheme to our table panel. I have reopened the issue and edited the title. It is always helpful if you provide as much information as possible

**Comment by Gerben-T** [comment]
I used the [Statusmap plugin](https://grafana.com/grafana/plugins/flant-statusmap-panel).
Here you can define your own Colors
![image](https://user-images.githubusercontent.com/26112463/109939363-4609f980-7cd1-11eb-96c1-3846de846afa.png)

**Comment by pkolyvas** [comment]
We're in the process of moving Grafana feature requests to [GitHub Discussions](https://community.grafana.com/t/were-moving-grafana-feature-requests-to-github-discussions/50341/3). This issue is being moved to [Discussions](https://github.com/grafana/grafana/discussions) or has been moved from our issues list, depending on where you're seeing this message. 

 We hope to use the discussion thread to continue building use cases, and additional evidence based on engagement/voting, in order to help us prioritize this feature request later on down the road. 

As with all things Grafana, we value your input and participation in this project and appreciate your patience during this transitional period. Looking forward to the discussion!

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
